layouts/partials/head.html

```diff
@@ -1,9 +1,25 @@
 <head>
-	<title>{{ .Site.Title }}</title>
+	<title>About the IPFS.io Public Gateway</title>
 	<meta charset="utf-8" />
```

Comment:
Why are you hardcoding this here? Please move this to the config, and leave as before, so that it's easier to migrate the site to a new design later if needed.

Reply:
Fixed in 8e02500c15ac9f2cb05e848f5f94bec720536499